hera_cal/frf.py

```diff
@@ -16,11 +16,85 @@
 from pyuvdata import UVData, UVFlag
 import argparse
 from . import io
+from . import vis_clean
 import warnings
+import astropy.constants as const
 
 
-def timeavg_waterfall(data, Navg, flags=None, nsamples=None, wgt_by_nsample=True, 
-                      wgt_by_favg_nsample=False, rephase=False, lsts=None, freqs=None, 
+SPEED_OF_LIGHT = const.c.si.value
+SDAY_KSEC = 86163.93 / 1000.
+
+
+def sky_frates(uvd, keys=None, frate_standoff=0.0, frate_width_multiplier=1.0, min_frate_half_width=0.025):
+    """Automatically compute sky fringe-rate ranges based on baselines and telescope location.
```

Comment:
Compute sky fringe-rate....baselines, telescope location, and frequencies in uvdata

Reply:
fixed.